src/sonic-yang-models/yang-models/sonic-vxlan.yang

```diff
@@ -0,0 +1,205 @@
+        }
+
+        container VXLAN_TUNNEL_MAP {
+
+            list VXLAN_TUNNEL_MAP_LIST {
+                key "name mapname";
+
+                leaf name {
+                    type leafref {
+                        path "../../../VXLAN_TUNNEL/VXLAN_TUNNEL_LIST/name";
+                    }
+                }
+
+                leaf mapname {
+                    type string;
+                }
+
+                leaf vlan {
+                    mandatory true;
+                    type leafref {
+                        path "/svlan:sonic-vlan/svlan:VLAN/svlan:VLAN_LIST/svlan:name";
+                    }
+
+                }
+
+                leaf vni {
+                    mandatory true;
+                    type uint32 {
+                        range "1..16777215" {
+                            error-message "Error: VNI ID out of range";
+                            error-app-tag vnid-invalid;
+                        }
+                    }
+                }
+            }
+        }
+
+        container VXLAN_EVPN_NVO {
+
+            list VXLAN_EVPN_NVO_LIST { 
+
+                key "name";
+                max-elements 1;
+
+                leaf name {
+                    type string;
+                }
+
+                leaf source_vtep {
+                    mandatory true;
+                    type leafref {
+                        path "../../../VXLAN_TUNNEL/VXLAN_TUNNEL_LIST/name";
+                    }
+                }
+            }
+        }
+        container VXLAN_TUNNEL_TABLE { 
+            sonic-ext:db-name "STATE_DB"; 
+            sonic-ext:key-delim ":"; 
+
+            config false;
+            description "state db vxlan tunnel table";
+
+            list VXLAN_TUNNEL_TABLE_LIST { 
+                key "name";
+
+                leaf name {
+                    type string; 
+                }
+
+                leaf src_ip {
+                    type string;
+                }
+
+                leaf dst_ip {
+                    type string;
+                }
+
+                leaf tnl_src {
+                    type string;
+                }
+
+                leaf operstatus {
+                    type string;
+                }
+            }
+        }
+        container VXLAN_REMOTE_VNI_TABLE {
+            sonic-ext:db-name "APPL_DB";
+            sonic-ext:key-delim ":";
+
+            config false;
+            description "app db vxlan remote vni table";
+
+            list VXLAN_REMOTE_VNI_TABLE_LIST {
+                key "vlan remote_vtep";
+
+                leaf vlan {
+                    type leafref {
+                        path "/svlan:sonic-vlan/svlan:VLAN/svlan:VLAN_LIST/svlan:name";
+                    }
+                }
+
+                leaf remote_vtep {
+                    type inet:ipv4-address;
+                }
+
+                leaf vni {
+                    type string;
+                }
+            }
+        }
+        container VXLAN_FDB_TABLE{
```

Comment:
I can't find UT for this table

Reply:
Do we have any infra support for this ? I see a sample_config_db.json but not a sample_app_db or sample_state_db..

Reply:
No support yet. Could you add this support?

Reply:
I have removed the APP and STATE DB table changes and kept only the CONFIG DB changes. Will add the remaining once the support for these tables comes in in the framework.